src/methods.js

```diff
@@ -1,111 +1,153 @@
 /* eslint-disable no-inline-comments */
 
 /**
+ * Module dependencies.
+ */
+
+import { map, set } from 'lodash';
+
+/**
+ * Obfuscating string.
+ */
+
+const obfuscated = '******';
```

Comment:
Why a const here but on the other other file? And this should either be uppercase or the value used directly.